README.md

```diff
@@ -32,7 +32,7 @@ Differing from the current official [Nix](https://github.com/NixOS/nix) installe
 * In `nix.conf`:
   + the `auto-allocate-uids`, `nix-command` and `flakes` features are enabled
   + `bash-prompt-prefix` is set
-  + `auto-optimise-store` is set to `true`
+  + `auto-optimise-store` is set to `true` (On Linux only)
   * `extra-nix-path` is set to `nixpkgs=flake:nixpkgs`
   * `auto-allocate-uids` is set to `true`.
```

Comment:
While we're here, I think auto uid is also linux-only? (and above when listing the features we enable)

Reply:
Oh gosh you're right!